packages/jsii/test/enums.test.ts

```diff
@@ -57,3 +57,16 @@ test('enums can have a mix of letters and number', async () => {
     { name: 'IB3M' },
   ]);
 });
+
+test('enums with the same assigned value should fail', async () => {
```

Comment:
Good!

But can you also add a `negative` test case? Those are used to validate the diagnostic message is bound to the proper context & looks actionnable from the user perspective.

Reply:
✅